app/src/docs/v1.api-spec.yaml

```diff
@@ -762,6 +793,38 @@ paths:
           $ref: "#/components/responses/Forbidden"
         default:
           $ref: "#/components/responses/Error"
+  /object/{objectId}/sync:
+    get:
+      summary: Synchronize an existing object
+      description: |-
+        Synchronize an existing COMS object so that any changes made externally (e.g. using an S3 client
+        to upload a new version) are tracked by COMS.
+
+        The object is added to an internal processing queue; once it is eventually processed,
+        the changes are tracked by COMS.
```

Comment:
Requests to enqueue a synchronization operation for a specific COMS object.

Reply:
Editing second paragraph to the following, as discussed earlier:
> This endpoint does not guarantee immediately synchronized results. Synchronization latency will be affected by the remaining number of objects awaiting synchronization.